screens/ChatsScreen.js

```diff
@@ -146,17 +107,17 @@ class ChatsScreen extends React.Component {
     }
 }
 
-function mapStateToProps({ chats }) {
-    return { chats };
-}
-
-const mapDispatchToProps = {
-    createMessage,
-    setCurrentUser,
-    setCurrentChat,
-    createChat
+const mapStateToProps = ({ chatsReducer: { chats } }) => {
```

Comment:
if you pass state here, it gets logged as { chatsReducer: {...}, messageReducer: {...}} etc, so you need to destructure that here to get at "chats"